src/components/shift-schedule/AdministratorMenu.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import { Button } from '@/components/ui/Button';
+import {
+  Collapsible,
+  CollapsibleContent,
+  CollapsibleTrigger,
+} from '@/components/ui/Collapsible';
+import { ChevronDownIcon, ChevronUpIcon, Trash2Icon } from 'lucide-react';
+import { useState } from 'react';
+
+type AdministratorMenuProps = {
+  t: {
+    label: string;
+    open: string;
+    close: string;
+    clearShiftSchedule: string;
+  };
+};
+
+function AdministratorMenu({ t }: AdministratorMenuProps) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <Collapsible
+      open={isOpen}
+      onOpenChange={setIsOpen}
+      className='mx-auto xs:mx-8 mt-8 rounded border p-3'
+    >
+      <div className='mx-1 flex justify-between'>
+        <span className='my-auto font-semibold text-xl'>{t.label}</span>
+        <CollapsibleTrigger asChild>
+          <Button variant='ghost'>
+            {isOpen ? (
+              <ChevronUpIcon aria-label={t.close} className='size-4' />
```

Comment:
put the aria label on the button instead of on the icon, then you can put aria-hidden="true" on the icons. This is because the button is the interactive element that will be selected and read by the screen reader